apps/console/src/features/applications/data/application-templates/templates/m2m-application/m2m-application.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Do we need to add this to https://github.com/wso2/carbon-identity-framework/tree/master/features/extension-mgt/org.wso2.carbon.identity.extension.mgt.feature/resources/extensions/applications as well?

Reply:
I did not find a usage of adding these resources here.